jenkins/janitor.py

```diff
@@ -110,9 +110,11 @@ def clear_resources(project, col, resource):
             base.append('--%s=%s' % (resource.condition, col))
 
         print "Try to kill %s - %s" % (col, list(items))
-        if subprocess.call(base + list(items)) != 0:
+        try:
+            subprocess.call(base + list(items))
+        except subprocess.CalledProcessError, e:
             err = 1
-            print "Error try to delete %s - %s" % (col, list(items))
```

Comment:
Thanks! note that most python code (including in our repo, generally at Google) tends to use single-quote rather than double-quote.